cmd/kops/create_secret_cacert.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/golang/glog"
+	"github.com/spf13/cobra"
+	"k8s.io/kops/cmd/kops/util"
+	"k8s.io/kops/pkg/apis/kops/registry"
+	"k8s.io/kops/upup/pkg/fi"
+	"k8s.io/kops/upup/pkg/fi/utils"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	"k8s.io/kubernetes/pkg/util/i18n"
+)
+
+var (
+	create_secret_cacert_long = templates.LongDesc(i18n.T(`
+	Add a intermediate ca certificate key private key. and store the key
+	in state store.`))
+
+	create_secret_cacert_example = templates.Examples(i18n.T(`
+	# Add a intermediate ca certificate key private key. and store the key
+	kops create secret ca-cert \
```

Comment:
We have environments that create k8s clusters and then tear them down.  This allows developers, consultants, R/D teams to create clusters for use cases and then tear them down.   Each time, the CA and signed cert is unique and causes extra clicks to success for browsers and clients to the API server (e.g. dashboard).   I share this as it may help with use cases.